snuba/migrations/validator.py

```diff
@@ -111,7 +111,8 @@ def _validate_order_new(
                 if op.target == OperationTarget.LOCAL:
                     _validate_add_col_or_create_table(op, dist_ops_before)
             elif isinstance(op, DropColumn):
-                _validate_drop(op, local_ops_before)
+                if op.target == OperationTarget.DISTRIBUTED:
+                    _validate_drop(op, local_ops_before)
```

Comment:
Let's rename the method to make it clear that it is validating drop column ordering. For a moment I thought it was validating drop tables.
```suggestion
                    _validate_drop_col(op, local_ops_before)
```

Reply:
fixed